Classes/Networking/RequestBody/TMGZIPEncodedRequestBody.m

```diff
@@ -0,0 +1,119 @@
+//
+//  TMGZIPEncodedRequestBody.m
+//  TMTumblrSDK
+//
+//  Created by Michael Benedict on 2/9/18.
+//
+
+#import "TMRequestBody.h"
+#import "TMGZIPEncodedRequestBody.h"
+#include <zlib.h>
+#define CHUNKSIZE (1024*4)
+
+@interface TMGZIPEncodedRequestBody ()
+@property (nonatomic, nonnull, readonly) id<TMRequestBody> originalBody;
+@property (nonatomic, nonnull, readonly) NSData *compressedBodyData;
+-(NSData*)compressedBody:(NSData *)data;
```

Comment:
New lines between the interface delaration and the properties and the @end, and you don't need to declare this method here